assets/package.json

```diff
@@ -28,13 +25,15 @@
   "devDependencies": {
     "@types/dom-screen-wake-lock": "^1.0.0",
     "@types/phoenix": "^1.5.4",
+    "@types/ramda": "^0.29.0",
     "@types/react": "^18.0.21",
     "@types/react-dom": "^18.0.6",
     "@types/react-page-visibility": "^6.4.1",
     "@types/uuid": "^8.3.4",
     "@typescript-eslint/eslint-plugin": "^5.40.1",
     "@typescript-eslint/parser": "^5.40.1",
     "autoprefixer": "^10.4.13",
+    "date-fns": "^2.29.3",
```

Comment:
Shouldn't that be a regular dependency? (i.e. not dev)

Reply:
Yeaaah, it should